utils/build/docker/java/spring-boot/src/main/java/com/datadoghq/system_tests/springboot/debugger/Pii.java

```diff
@@ -0,0 +1,94 @@
+package com.datadoghq.system_tests.springboot;
+
+public class Pii {
+    public static String VALUE = "SHOULD_BE_REDACTED";
+    public String TestValue = VALUE;
+
+    public String _2fa = VALUE;
+    public String accesstoken = VALUE;
```

Comment:
What could be also tested is the fact that if the keyword is in uppercase or with separators, we normalize before testing against the keyword list

Reply:
Hmmm, not sure about it. I'm not testing the inner implementation of redaction mechanism, just input vs output.
I added some non redacted fields here 2b38a79f301c78230dc3ec78c8ddd0f87cb8f900 is it good enough?

Reply:
I understand the fact that you are not testing the inner workings, but verify that `ACCESSTOKEN` or `Access_Token` are readacted as well than `accesstoken`.

Reply:
Added here 6e77088af8c173f8306b511b6fe521640923c8b5